php/ubiquity/app/cache/controllers/routes.default.cache.php

```diff
@@ -0,0 +1,2 @@
+<?php
```

Comment:
can we **NOT** create `cache` on container ?

Reply:
I don't understand the question.
Do you want me to remove this file?
It is automatically generated when the container is created